python/lsst/ts/m2com/mock/mock_server.py

```diff
@@ -280,9 +284,14 @@ async def _send_welcome_message(self) -> None:
         await self._message_event.write_hardpoint_list(hardpoints)
 
         await self._message_event.write_interlock(False)
-        await self._message_event.write_inclination_telemetry_source(
-            self.model.inclination_source
-        )
+
+        # Workaround of the mypy checking
+        is_external_source = self.model.control_parameters[
+            "use_external_elevation_angle"
+        ]
+        assert type(is_external_source) is bool
```

Comment:
See my comment above, you might be able to remove this `assert` here if you use `is True` as I mentioned above.